exercises/list-ops/src/test/java/ListOpsTest.java

```diff
@@ -303,62 +282,158 @@ public void shouldReduceWithAnticommutativeAccumulator() {
                 result.add(elem);
                 return result;
             };
-
+    
     private BinaryOperator<List<Integer>> combiner
             = (list1, list2) -> {
                 List<Integer> result = new ArrayList<>(list1);
                 result.addAll(list2);
                 return result;
             };
-
+    
     @Test
     @Ignore
     public void shouldReduceAnEmptyListAndANonEmptyListAndReturnConcatenation() {
         final List<Integer> list = Collections.unmodifiableList(
-                IntStream.range(0, 5)
-                        .boxed()
-                        .collect(Collectors.toList())
+                Arrays.asList(0, 1, 2, 3, 4, 5)
         );
         final List<Integer> actual
                 = ListOps.reduce(list,
                         new ArrayList<Integer>(),
                         accumulator,
                         combiner);
         final List<Integer> expected
-                = IntStream.range(0, 5)
-                        .boxed()
-                        .collect(Collectors.toList());
-
+                = Arrays.asList(0, 1, 2, 3, 4, 5);
+        
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
-
+    
     @Test
     @Ignore
     public void shouldReduceTwoNonEmptyListsAndReturnConcatenation() {
         final List<Integer> listOne = Collections.unmodifiableList(
-                IntStream.range(0, 5)
-                        .boxed()
-                        .collect(Collectors.toList())
+                Arrays.asList(0, 1, 2, 3, 4)
         );
         final List<Integer> listTwo = Collections.unmodifiableList(
-                IntStream.range(5, 10)
-                        .boxed()
-                        .collect(Collectors.toList())
+                Arrays.asList(5, 6, 7, 8, 9)
         );
         final List<Integer> actual
                 = ListOps.reduce(listTwo,
                         listOne,
                         accumulator,
                         combiner);
         final List<Integer> expected
-                = IntStream.range(0, 10)
-                        .boxed()
-                        .collect(Collectors.toList());
+                = Arrays.asList(0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
+        
+        assertNotNull(actual);
+        assertFalse(actual.isEmpty());
+        assertEquals(expected, actual);
+    }
+    
+    @Test
+    @Ignore
+    public void shouldReturnZeroWhenAnEmptyListIsCollected() {
```

Comment:
😱  OH GEEEZ!  More scope?!?! :)

This is already the longest running PR I've seen on this track.  I strongly prefer we first land this thing before adding any more to the exercise.

It opens a can of worms as to whether to include both `reduce` and `collect`; the two are very similar.  And it duplicates the concern of invoking the notion of parallelizable processing of stream operations.

Would you mind stashing this work somewhere?  I suspect we'll break this thing up into chunks and it won't seem so overwhelming... later.